.github/workflows/build.yaml

```diff
@@ -161,4 +154,6 @@ jobs:
           cache-from: |
             ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}/solr:${{ env.TAG }}
           push: true
-          tags: ${{ steps.meta-solr.outputs.tags }}
+          tags: |
+            ${{ steps.meta-solr.outputs.tags }}
+            ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}/solr:${{ env.TAG }}
```

Comment:
When we push the image to the registry it will include:
The metadata tags: `${{ steps.meta-solr.outputs.tags }}`
and the short sha from `${{ env.REGISTRY }}/${{ env.REPO_LOWER }}/solr:${{ env.TAG }}` is added as well. The latest tag will only happen when on the default branch.